src/styles.css

```
@@ -202,11 +202,20 @@ body{background:var(--bg);color:var(--ink);font-family:'Plus Jakarta Sans',sans-
 .bookmark-btn:focus-visible{
   outline:2px solid var(--orange);
   outline-offset:2px;
 }
+.bookmark-btn:hover {
+  transform: scale(1.2);
+  box-shadow: 0 0 10px rgba(245, 158, 11, 0.7);
+}
+
+.bookmark-btn:active {
+  transform: scale(0.9);
+}ssssssssss
```

Stylelint (17.11.0)

[error] Unknown type selector "ssssssssss" (selector-type-no-unknown)

(selector-type-no-unknown)